GUI/src/main/kotlin/ui/components/general/ProjectManager.kt

```diff
@@ -59,18 +45,20 @@ fun projectMenu(
             expanded = expanded,
             onDismissRequest = { expanded = false },
         ) {
+            val openScope = rememberCoroutineScope()
             DropdownMenuItem(
                 onClick = {
-                    openFileChooserAndGetPath()?.let { handleOpenProject(state, it) }
+                    openScope.launch(Dispatchers.IO) { openFileChooserAndGetPath { path -> handleOpenProject(state, path) } }
```

Comment:
The async'ness here allows the user to navigate a splitsecond before the filepicker is loaded. Example is press the Save Collage button, then quickly press the right arrow key (on your keyboard) and then the picker opens.
This will probably lead to unexpected behaviour although it is only a very edge case. 
We should think about not making this async, as we dont actually want the user to interacted while the picker is not opened up. Otherwise, we could do it async but lock the main thread somehow xD

Reply:
resolved by removing async from save collage and save png